pkg/cmd/server/server.go

```diff
@@ -144,6 +147,14 @@ func NewCommand() *cobra.Command {
 	command.Flags().StringSliceVar(&config.restoreResourcePriorities, "restore-resource-priorities", config.restoreResourcePriorities, "desired order of resource restores; any resource not in the list will be restored alphabetically after the prioritized resources")
 	command.Flags().StringVar(&config.defaultBackupLocation, "default-backup-storage-location", config.defaultBackupLocation, "name of the default backup storage location")
 
+	volumeSnapshotLocations := flag.NewMap()
+	volumeSnapshotLocations = volumeSnapshotLocations.WithKeyValueDelimiter(":")
+	command.Flags().Var(&volumeSnapshotLocations, "default-volume-snapshot-locations", "provider map of the default volume snapshot locations")
```

Comment:
"map of each volume provider's default volume snapshot location"

Reply:
I always try to avoid using name of a data structure, even if it totally fits. Could we say instead: `list of unique volume providers and corresponding default volume snapshot location`?